Sources/CoreMI/Intramodular/Service/_MIServiceTypeIdentifier.swift

```diff
@@ -38,4 +38,6 @@ extension _MIServiceTypeIdentifier {
     public static let _VoyageAI = _MIServiceTypeIdentifier(rawValue: "hajat-fufoh-janaf-disam")
     public static let _Cohere = _MIServiceTypeIdentifier(rawValue: "guzob-fipin-navij-duvon")
     public static let _TogetherAI = _MIServiceTypeIdentifier(rawValue: "pafob-vopoj-lurig-zilur")
+    public static let _PlayHT = _MIServiceTypeIdentifier(rawValue: "foluv-jufuk-zuhok-hofid")
+    public static let _Rime = _MIServiceTypeIdentifier(rawValue: "tohaz-zivir-bosov-minog")
```

Comment:
@Archetapp did you generate these using `HadeanIdentifier.random()`?

Reply:
@vmanot I did yes.